index.d.ts

```diff
@@ -101,6 +101,26 @@ export interface IGetOrderStatusResponse {
   orderId: number;
 }
 
+export interface IGetOrdersRequest {
+  token: string;
+  dateFilterMin?: string;
+  dateFilterMax?: string;
+  statusFilter?: number;
+  page?: number;
+}
+
+export interface IGetOrder {
```

Comment:
This interface describes a custom/partial representation of an order and not really an action. Could you rename it by something like `IOrderSummary`?